shaman/controllers/root.py

```diff
@@ -1,8 +1,20 @@
 from pecan import expose
 
+from shaman.controllers import repos
+
 description = "shaman is the source of truth for the state of repositories on chacra nodes."
 
 
+class APIController(object):
+    @expose('json')
+    def index(self):
+        return dict(
+            repos="",
```

Comment:
This is just a placeholder currently and shows the user what endpoint was next. Sure, that data seems like it might be useful, I hadn't put much thought into that specific part of this PR.